src/rc/start-stop-daemon.c

```diff
@@ -96,6 +100,7 @@ const struct option longopts[] = {
 	{ "stderr",       1, NULL, '2'},
 	{ "stdout-logger",1, NULL, '3'},
 	{ "stderr-logger",1, NULL, '4'},
+	{ "capabilities", 1, NULL, 'B'},
```

Comment:
why `-B` ?  is that a common short flag in tools that i'm not aware of ?  seems like `-C` would make more sense since "capabilities" starts with "c" (and `-c` is already taken).

Reply:
`C` is also already taken, by `_usage.h`. Which letter should I use?

Reply:
I used `B` again.

Reply:
thanks, i was only looking at local letters

let's not allocate a short option for it at all.  use 0x80 in the long opts array so you can look it up the table.

Reply:
What do mean? I am not an expert about getopt framework? How I make it only long option?

Reply:
I think it just means replacing all the relevant instances of `'B'` with `0x80`, because it's out of the range of ascii and so won't get passed on the command line, but can still be checked in the switch.

Reply:
sorry for not being clear. what @E5ten suggested is good.

Reply:
Okay, I figured it out. @E5ten Thanks for your help.